cli.js

```diff
@@ -121,6 +121,12 @@ program
 
 							writeStream.write((wroteData ? '\n' : '') + data.join('\n'));
 							break;
+
+						case 'stdout':
+							data = _.map(data, function(row) {
+								console.log(row.ipAddress + ':' + row.port);
+							});
```

Comment:
If you take a look at the implementation of `console.log`, you see that it just wraps `stdout.write`, appends an *\n* and allows formatted output.

But I can change it to `stdout.write` if you prefer...